### DISCUSSION TITLE
Can I insert excel in DXF?

### DISCUSSION THREAD

**Discussion by liunx7594** [mozman/ezdxf#418] [Q&A]
Can I insert excel in DXF? As far as I know, it seems that the ole2frame structure of CAD can store it.

I tried to put it into a block and insert another DXF file, but it didn't work.
I used the following code

```
import ezdxf
from ezdxf.addons import Importer

cad1 = ezdxf.readfile(r'1.dxf')
cad2 = ezdxf.readfile(r'2.dxf')

msp2 = cad2.modelspace()

importer = Importer(cad2, cad1)
importer.import_block('b_test')
importer.finalize()
msp1 = cad1.modelspace()
msp1.add_blockref('b_test', insert=(10, 10))
cad1.save()
```

I uploaded these two files

[dxf.zip](https://github.com/mozman/ezdxf/files/6383493/dxf.zip)

**Comment by mozman** [comment]
In DXF yes as OLE object, but without any support by ezdxf.
